src/sql/parts/notebook/models/notebookContexts.ts

```diff
@@ -8,10 +8,11 @@
 import { nb } from 'azdata';
 
 import { localize } from 'vs/nls';
-import { IDefaultConnection, notebookConstants } from 'sql/parts/notebook/models/modelInterfaces';
+import { IDefaultConnection } from 'sql/parts/notebook/models/modelInterfaces';
 import { IConnectionManagementService } from 'sql/platform/connection/common/connectionManagement';
 import { ConnectionProfile } from 'sql/platform/connection/common/connectionProfile';
 import { IConnectionProfile } from 'sql/platform/connection/common/interfaces';
+import { sqlKernelSpec } from 'sql/workbench/services/notebook/sql/sqlSessionManager';
```

Comment:
Minor: could we move this kernelSpec to modelInterfaces? I don't like this code knowing about the sqlSessionManager directly. It encourages us to bind directly to that file whereas we might want to move it to an extension in the long term.

Reply:
This is good idea